proposals/improved-interpolated-strings.md

```diff
@@ -35,16 +35,16 @@ convenient interpolation syntax.
 We introduce a new builder pattern that can represent an interpolated string passed as an argument to a method. The simple English of the pattern is as follows:
 
 When an _interpolated\_string\_expression_ is passed as an argument to a method, we look at the type of the parameter. If the parameter type has a static method
-`GetInterpolatedStringBuilder` that can invoked with 2 int parameters, `baseLength` and `formatHoleCount`, optionally takes a parameter the receiver is convertible to,
-and has an out parameter of the type of original method's parameter and that type has instance `TryFormat` methods can be invoked for every part of the interpolated
+`Create` that can invoked with 2 int parameters, `baseLength` and `formatHoleCount`, optionally takes a parameter the receiver is convertible to,
+and has an out parameter of the type of original method's parameter and that type has instance `TryFormatX` methods can be invoked for every part of the interpolated
```

Comment:
>  If the parameter type has a static method `Create` that can invoked with 2 int parameters

This will impact the static virtual interfaces proposal as type parameters will become eligable in certain cases to be interpolated builders. Can we add a tracking issue in the proposal that we need to handle this at some point in the future.

Don't want to design it, just want to make sure we are tracking that these features overlap in this space.